README.md

```diff
@@ -37,43 +37,51 @@ at <a href="https://docs.snowflake.net/manuals/user-guide/nodejs-driver.html">No
 Test
 ======================================================================
 
-Prepare for Test
+Prepare for Tests
 ----------------------------------------------------------------------
 
-Set the Snowflake connection info in ``parameters.json`` and place it in $HOME:
-<pre><code>
-{
-  "testconnection": {
-    "SNOWFLAKE_TEST_USER":      "&lt;your_user&gt;",
-    "SNOWFLAKE_TEST_PASSWORD":  "&lt;your_password&gt;",
-    "SNOWFLAKE_TEST_ACCOUNT":   "&lt;your_account&gt;",
-    "SNOWFLAKE_TEST_WAREHOUSE": "&lt;your_warehouse&gt;",
-    "SNOWFLAKE_TEST_DATABASE":  "&lt;your_database&gt;",
-    "SNOWFLAKE_TEST_SCHEMA":    "&lt;your_schema&gt;",
-    "SNOWFLAKE_TEST_ROLE":      "&lt;your_role&gt;"
-  }
-}
-</code></pre>
+Specify env variables:
+
+```
+export SNOWFLAKE_TEST_USER=<your_user>
+export SNOWFLAKE_TEST_PASSWORD=<your_password>
+export SNOWFLAKE_TEST_ACCOUNT=<your_account>
+export SNOWFLAKE_TEST_WAREHOUSE=<your_warehouse>
+export SNOWFLAKE_TEST_DATABASE=<your_database>
+export SNOWFLAKE_TEST_SCHEMA=<your_schema>
+export SNOWFLAKE_TEST_PROTOCOL=<your_snowflake_protocol>
+export SNOWFLAKE_TEST_HOST=<your_snowflake_host>
+export SNOWFLAKE_TEST_PORT=<your_snowflake_port>
```

Comment:
Are there some default values? Maybe we should somehow mention it here, that these values are not required?

Reply:
There are no defaults.